### ISSUE TITLE
Feature Request: Implement: MODE CON: rate=32 delay=1

### ISSUE THREAD

**Issue by JDoucette** [robhagemans/pcbasic#164]
## Feature Request
I always run at the DOS command prompt:
```
MODE CON: rate=32 delay=1
```
To speed up the repeat rate to be as fast as possible (32 is about 30 characters/second), and to delay the least possible (1 = 0.25 second delay, out of 4 values: 1..4)
E.g. explanation webpage: https://www.palmtoppaper.com/ptphtml/50/pt500030.htm

This would be awesome to have as command line parameters! :)

**Comment by rbergen** [comment]
I'm not sure this is possible. I've checked the code involved with keyboard input, and there is no code introducing or managing delay or repetition rate. Testing has shown me that on Windows, the repetition delay and rate are determined by the rates configured using the Keyboard applet in the Control Panel:
![image](https://user-images.githubusercontent.com/10425218/141610833-64583763-3c1f-4208-bc31-6b8fb3b90a6a.png)

That is to say, repetition delay and rate are determined by the operating system PC-BASIC runs on, which is actually exactly how it works in GW-BASIC. This is thus the correct behavior from an emulation standpoint.

**Comment by JDoucette** [comment]
I just tested with PC-BASIC 2.0 and this is no longer an issue.

For reference, I have the settings shortest delay and fastest repeat, and PC-BASIC 1.2 ignores this, but PC-BASIC 2.0 respects this.

P.S. I'll test the other bugs I've reported with PC-BASIC 2.0 as well.  I was downloading the latest version here:
https://sourceforge.net/projects/pcbasic/
Which links to 1.2.

From now on, I will use your official site:
http://robhagemans.github.io/pcbasic/

Again... thanks immensely for making this emulator.  I've been meaning to run my old programs for years!
